existing installs break on upgrade from the artifactory.admin keys

seems there is a permission error on existing installs with bootstrap creds

```
Caused by: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'jerseyConfig' defined in URL [jar:file:/opt/jfrog/artifactory/app/artifactory/tomcat/webapps/access/WEB-INF/lib/access-application-7.99.6.jar!/org/jfrog/access/rest/config/JerseyConfig.class]: Bean instantiation via constructor failed; nested exception is org.springframework.beans.BeanInstantiationException: Failed to instantiate [org.jfrog.access.rest.config.JerseyConfig]: Constructor threw exception; nested exception is org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'projectResource' defined in URL [jar:file:/opt/jfrog/artifactory/app/artifactory/tomcat/webapps/access/WEB-INF/lib/access-server-rest-7.99.6.jar!/org/jfrog/access/server/rest/resource/project/ProjectResource.class]: Unsatisfied dependency expressed through constructor parameter 0; nested exception is org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'projectServiceImpl' defined in URL [jar:file:/opt/jfrog/artifactory/app/artifactory/tomcat/webapps/access/WEB-INF/lib/access-server-core-7.99.6.jar!/org/jfrog/access/server/service/projects/ProjectServiceImpl.class]: Unsatisfied dependency expressed through constructor parameter 12; nested exception is org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'entitlementsService' defined in URL [jar:file:/opt/jfrog/artifactory/app/artifactory/tomcat/webapps/access/WEB-INF/lib/access-server-core-7.99.6.jar!/org/jfrog/access/server/service/license/EntitlementsService.class]: Unsatisfied dependency expressed through constructor parameter 0; nested exception is org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'jfConnectConfiguration' defined in URL [jar:file:/opt/jfrog/artifactory/app/artifactory/tomcat/webapps/access/WEB-INF/lib/access-server-core-7.99.6.jar!/org/jfrog/access/server/service/jfconnect/JfConnectConfiguration.class]: Unsatisfied dependency expressed through constructor parameter 0; nested exception is org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'systemServiceImpl' defined in URL [jar:file:/opt/jfrog/artifactory/app/artifactory/tomcat/webapps/access/WEB-INF/lib/access-server-core-7.99.6.jar!/org/jfrog/access/server/service/system/SystemServiceImpl.class]: Unsatisfied dependency expressed through constructor parameter 5; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'accessServerBootstrapImpl': Invocation of init method failed; nested exception is java.lang.RuntimeException: The '/opt/jfrog/artifactory/var/etc/access/bootstrap.creds' are not as expected. Expected permissions are: OWNER_READ OWNER_WRITE while the current permissions are: OWNER_WRITE GROUP_WRITE GROUP_READ OWNER_READ.
```

--- a/README.md
+++ b/README.md
@@ -46,6 +46,20 @@ The released packages can be found in [ghcr](https://github.com/defenseunicorns/
 > [!TIP]
 > To get a list of tasks to run you can use `uds run --list`!
 
+## Redeploying/Upgrading pre-existing Artifactory
+
+> [!IMPORTANT]
+> **NOTE**: If admin credentials are changed or different from what is generated in the `artifactory-admin-credentials` secret, in the `values/common.yaml` set:
+
+``` yaml
+artifactory:
+  admin:
+    secret: ""
+    dataKey: ""
+```
+
+The credentials will be reset to what is in the secret if not set to empty or might break the deployment/upgrade entirely.
+
 ## Contributing
 
 Please see the [CONTRIBUTING.md](./CONTRIBUTING.md)

--- a/values/common.yaml
+++ b/values/common.yaml
@@ -3,7 +3,7 @@ artifactory:
     secret: artifactory-admin-credentials
     dataKey: adminCreds
   openMetrics:
-    enabled: false
+    enabled: true
 database:
   type: postgresql
   driver: org.postgresql.Driver

